src/shared/workers/serviceWorker.ts

```diff
@@ -14,3 +14,21 @@ export const registerServiceWorker = (): Promise<ServiceWorkerRegistration> => {
 
   return workerRegistration
 }
+
+export const registerServiceWorkerInfluxQL =
+  (): Promise<ServiceWorkerRegistration> => {
+    // Worker -- load prior to page load event, in order to intercept fetch in http/2.
+    // see service worker life cycle. https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers
+    let workerRegistrationInfluxQL = null
+    if ('serviceWorker' in navigator) {
+      workerRegistrationInfluxQL = navigator.serviceWorker.register(
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        new URL('../workers/downloadHelper.ts', import.meta.url),
+        {scope: '/query'}
```

Comment:
Adding another service worker because the `scope` of InfluxQL `/query` is different from that of Flux `/api/v2/query`